configs/lgbm_mb_cpu_config.json

```diff
@@ -1,109 +1,120 @@
 {
-    "omp_env": ["OMP_NUM_THREADS", "OMP_PLACES"],
     "common": {
-        "lib": ["modelbuilders"],
-        "data-format": ["pandas"],
-        "data-order": ["F"],
-        "dtype": ["float32"]
+        "lib":          "modelbuilders",
```

Comment:
Add note to README that parameters might be set with single value or list of values

Reply:
done earlier